CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file.
 
+## 4.1.0 - 2023-10-09
+
+### Added
+
+- `isPasswordConfirmationRequired(): boolean` function to replace deprecated `OC.PasswordConfirmation.requiresPasswordConfirmation()`
+
+### Fixed
+
+- Improve password confirmation dialog design and accessibility
+- Fix compatibility with `@nextcloud/l10n@^2`
+
+### Changed
+
+- Major update `@nextcloud/vue` to v8.0.0-beta.7
```

Comment:
from https://nextcloud-vue-components.netlify.app/

> [Latest version: v8.x (Nextcloud 28)](https://nextcloud-vue-components.netlify.app/)
>     [next v9.x (Nextcloud 28 + Vue 3)](https://next--nextcloud-vue-components.netlify.app/)
>     [v7.x (Nextcloud 25 - 27)](https://stable7--nextcloud-vue-components.netlify.app/)
>     [v6.x (Nextcloud 24 - 25)](https://stable6--nextcloud-vue-components.netlify.app/)

if this component no longer works with 27 the release should be a major

Reply:
True. Should be 5.0.0 then

Reply:
Is there anything breaking on `nextcloud/vue@8` running under `server@27`?

Reply:
Design looking inconsistent and it will increase the bundle size a lot.